accounts/abi/argument.go

```diff
@@ -275,7 +275,7 @@ func capitalise(input string) string {
 	if len(input) == 0 {
 		return ""
 	}
-	return strings.ToUpper(input[:1]) + input[1:]
+	return toCamelCase(strings.ToLower(input[:1]) + input[1:])
```

Comment:
I would prefer if you:

  * used `strings.split` with `_` as the splitting character;
  * capitalized all strings in the resulting slice;
  * `join`-ed the slices back together, which will take care of cases like `"__"`; and,
  * renamed this function `camelize`.